Gems/AtomLyIntegration/CommonFeatures/Code/Source/CoreLights/LightDelegateBase.h

```diff
@@ -82,12 +82,16 @@ namespace AZ
             // TransformNotificationBus::Handler overrides ...
             void OnTransformChanged(const Transform& local, const Transform& world) override;
 
+            LmbrCentral::ShapeComponentRequests* m_shapeBus = nullptr;
```

Comment:
suggestion: It might be an idea to leave this private and then add an accessor for it that's protected to limit derived classes being able to modify it unnecessarily